RELEASES.md

```diff
@@ -1,3 +1,129 @@
+Version 1.33.0 (2019-02-28)
+==========================
+
+Language
+--------
+- [You can now use the `cfg(target_vendor)` attribute.][57465] E.g.
+  `#[cfg(target_vendor="linux")] fn main() { println!("Hello Linux!"); }`
+- [You can now have multiple patterns in `if let` and `while let`
+  expressions.][57532] You can do this with the same syntax as a `match`
+  expression. E.g.
+  ```rust
+  enum Creature {
+    Crab(String),
+    Person(String),
+  }
+
+  fn main() {
+    let state = Creature::Crab("Ferrous");
+
+    if let Creature::Crab(name) | Creature::Person(name) {
+      println!("This creature's name is: {}", name);
+    }
+  }
+  ```
+- [You can now have irrefutable `if let` and `while let` patterns.][57535] Using
+  this feature will by default produce a warning as this behaviour can be
+  unintuitive. E.g. `if let _ = 5 {}`
+- [You can now use `let` bindings and pattern destructurcting in
+  constant functions.][57175]
+- [You can now specify multiple attributes in a `cfg_attr` attribute.][57332]
+  E.g. `#[cfg_attr(all(), must_use, optimize)]`
+- [You can now specify a specific alignment with the `#[repr(packed)]`
+  attribute.][57049] E.g. `#[repr(packed(2))] struct Foo(i16, i32);` is a struct
+  with an alignment of 2 bytes and a size of 6 bytes.
+- [You can now call unsafe constant functions.][57067] E.g.
+  ```rust
+  unsafe const fn foo() -> i32 { 5 }
+  const fn bar() -> i32 {
+    unsafe { foo() }
+  }
+  ```
+- [You can now import an item from a module as a `_`.][56303] This allows you to
+  import a trait's impls, and not have the name in the namespace. E.g.
+  ```rust
+  use std::io::Read as _;
+
+  // Allowed as there is only one `Read` in the module.
+  pub trait Read {}
+  ```
+- [`extern` functions will now abort by default when panicking.][55982]
```

Comment:
Perhaps mention that this removes one instance of undefined behaviour from Rust, since that's quite significant. Also maybe that this was attempted in the past but had to be reverted.